relevanceai/dataset_api/dataset.py

```diff
@@ -495,6 +497,28 @@ def all(
             show_progress_bar=show_progress_bar,
         )
 
+    def to_csv(self, filename: str):
+        documents = self.get_all_documents(self.dataset_id)
+        df = pd.DataFrame(documents)
+        df.to_csv(filename)
+
+    def read_csv(self, filename: str):
```

Comment:
Why don't you use `client.insert_csv` here?

Reply:
True